### DISCUSSION TITLE
Database of stars on Stellarium

### DISCUSSION THREAD

**Discussion by Char1234567** [Stellarium/stellarium#3596] [General]
Hi Community :)

Dose anyone know how to extract all the stars available on the Stellarium database?
I am specifically looking for the below attributes: I would like to export all of the below information in an Excel spreadsheet.

Astrological Constellation
IAU Designation
Celestial Coordinates
Classification
Type
Apparent Color
Luminosity (Suns)
Radius (Suns)
Distance from Earth (Light Years)

Any help / guidance would be greatly appreciated.

Thank you

**Comment by gzotti** [comment]
"Astrological constellation" may refer to ecliptical longitude, or what do you mean?  Astronomical Constellation is one of 88 regions in the sky defined by IAU. Computed from coordinates on the fly, there is a paper for that. See User Guide.
Rest: See open-source code and User Guide.

**Reply by Char1234567** [reply]
Can you please direct me to place where this is in the user guide.

Thanks

**Reply by kenneyrsw** [reply]
https://stellarium.org/files/guide.pdf
